docker/release/ubuntu14.04/Dockerfile

```diff
@@ -12,25 +11,23 @@ RUN locale-gen $LANG && update-locale
 # Install dependencies and clean up
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
-        libc6 \
-        libcurl3 \
-        ca-certificates \
-        libgcc1 \
-        libicu52 \
-        libssl1.0.0 \
-        libstdc++6 \
-        libtinfo5 \
-        libunwind8 \
-        libuuid1 \
-        zlib1g \
-        curl \
-        git \
+		apt-utils \
+		ca-certificates \		
```

Comment:
Minor comment: there are multiple trailing white spaces at some lines in all 3 files, for example, this one.